tinkerpop-iam-authenticator/3.7/src/main/java/cloud/localstack/LsStateKey.java

```diff
@@ -0,0 +1,15 @@
+package cloud.localstack;
+
+import cloud.localstack.auth.HeaderAuthenticator.HeaderNegotiator;
+import io.netty.util.AttributeKey;
+
+public final class LsStateKey {
+    private LsStateKey() {
+    }
+
+    /**
+     * The key for signature v4 of the current request
```

Comment:
```suggestion
     * The key for the requests authorization header
```
I think `The key for signature v4` is a bit confusing